lib/player.rb

```diff
@@ -0,0 +1,16 @@
+class Player
+    attr_reader :first_name, :last_name, :monthly_cost, :contract_length
+  
+    def initialize(first_name, last_name, monthly_cost, contract_length)
```

Comment:
Make sure that you are reading the specification carefully and make sure that your code is matching what the specification wants. For example, the player class should accept three arguments, and you've written it so it accepts four. This isnt something that we can change at will.